openfl/experimental/runtime/local_runtime.py

```diff
@@ -62,6 +66,124 @@ def ray_call_get(self) -> List[Any]:
         return clones
 
 
+def ray_group_assign(collaborators, collaborators_per_group=3, max_concurrency=1):
+    # This function takes the list of collaborators and the collaborators per group and assigns the collaborators to resource groups which share cuda context,
+    # it returns a list of GroupMembers, that is a utility class that manages the collaborator and in which group it is in. This maintians compatablility with current runtime execution without having to manage the groups manually.
+    
+    class GroupMember:
+        # Utility class that maintains compatability with runtime execution,
+        # It takes the collaborator_actor, and the name of a collaborator. it takes care of managing which group the collaborator is a member of. 
+        # It also assigns attributes for each funtion in the Collaborator interface in conjuntion with RemoteHelper to maintain compatability with RayExecutor.
+        def __init__(self, collaborator_actor, collaborator):
+            interface_module = importlib.import_module(

```

Comment:
Is it necessary to use `importlib` here? Could this be simplified to:

```
from openfl.experimental.interface import Collaborator
all_methods = [
                method
                for method in dir(Collaborator)
                if callable(getattr(Collaborator, method))
            ]
```

Reply:
Changed to from openfl.experimental.interface import Collaborator